panning-view/src/main/java/com/maelstrom/panning/PanningImageView.java

```diff
@@ -26,9 +26,9 @@ public PanningImageView (Context context, AttributeSet attr, int defStyle) {
 		super (context, attr, defStyle);
 		if (!isInEditMode()) {
 			readStyleParameters(context, attr);
+			mAttacher = new PanningImageViewAttacher (this, mPanningDurationInMs);
```

Comment:
`mAttacher` would be undefined if the `if` block doesn't execute. I think it would be better to initialize `mPanningDurationInMs` with the default value.